photon-client/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorTest.scala

```diff
@@ -56,24 +76,35 @@ class GameEstimatorTest extends SparkTestUtils with GameTestUtils {
    *   Here, we have to setup an intercept manually, otherwise GameEstimator learns only a dependence on the
    *   features.
    */
-  @Test
-  def simpleFixedEffectTest(): Unit = sparkTest("simpleFixedEffectTest") {
+  @Test(dataProvider = "trivialLabeledPoints")
+  def simpleHardcodedTest(labeledPoints: Seq[LabeledPoint]): Unit = sparkTest("simpleHardcodedTest") {
+
+    import org.apache.log4j.{Level, Logger}
+
+    Logger.getLogger("org").setLevel(Level.OFF)
+    Logger.getLogger("akka").setLevel(Level.OFF)
```

Comment:
We can remove - the logging was making it harder to see what I needed to see in the trace. I'm removing.